android/settings.gradle

```diff
@@ -7,6 +7,7 @@ pluginManagement {
 
 plugins {
     id("com.android.library").version("8.3.0").apply(false)
```

Comment:
This plugin version does not match the one defined in build.gradle, not sure which one will have precedence.

Reply:
fixed this, removed config from `settings.gradle`